tests/admin/test-class-admin-editor-specific-replace-vars.php

```diff
@@ -8,247 +8,250 @@
-//	 *
-//	 * @covers       WPSEO_Admin_Editor_Specific_Replace_Vars_Test::get_editor_specific_replacevars_for
-//	 */
-//	public function test_get_editor_specific_replacevars( $page_type, $expected ) {
-//		$this->assertEquals( $expected, $this->class_instance->get_editor_specific_replacevars_for( $page_type ) );
-//	}
-//
-//	/**
-//	 * Dataprovider function for the test: test_get_editor_specific_replacevars
-//	 *
-//	 * @return array With the $page_type and $expected variables.
-//	 */
-//	public function get_editor_specific_replacevars_provider() {
-//		$editor_specific_replace_vars = $this->class_instance->get_protected_editor_specific_replace_vars();
-//
-//		return array_map( function( $name, $replace_vars ) {
-//			return array( $name, $replace_vars );
-//		}, $editor_specific_replace_vars );
-//	}
-//
-//	/**
-//	 * Tests that get_editor_specific_replacevars works when there are no recommendations found.
-//	 *
-//	 * @covers WPSEO_Admin_Editor_Specific_Replace_Vars_Test::get_editor_specific_replacevars_for
-//	 */
-//	public function test_get_editor_specific_replacevars_non_existing() {
-//		$this->assertEquals( array(), $this->class_instance->get_editor_specific_replacevars_for( 'non-existing-replace-var' ) );
-//	}
-//
-//	/**
-//	 * Tests that get_editor_specific_replacevars works when a filter adds a non-array recommendation.
-//	 *
-//	 * @covers WPSEO_Admin_Editor_Specific_Replace_Vars_Test::get_editor_specific_replacevars_for
-//	 */
-//	public function test_get_editor_specific_replacevars_non_array() {
-//		$this->assertEquals( array(), $this->class_instance->get_editor_specific_replacevars_for( 'non-array' ) );
-//	}
-//
-//	/**
-//	 * Filter function for adding or changing replacement variables.
-//	 *
-//	 * @param array $replacevars The replacement variables before the filter.
-//	 *
-//	 * @return array The new editor_specific replacement variables.
-//	 */
-//	public function filter_editor_specific_replacevars( $replacevars = array() ) {
-//		$replacevars[ 'non-array' ] = 'non-array';
-//
-//		return $replacevars;
-//	}
-//
-//	/**
-//	 * Create and get a mocked WP_Post with a certain post_type.
-//	 *
-//	 * @param string $post_type The post type to give to the post.
-//	 *
-//	 * @return WP_Post A mocked post with the specified post_type.
-//	 */
-//	private function create_and_get_with_post_type( $post_type = 'post' ) {
-//		return self::factory()->post->create_and_get(
-//			array(
-//				'post_type' => $post_type,
-//			)
-//		);
-//	}
-//}
+class WPSEO_Admin_Editor_Specific_Replace_Vars_Test extends WPSEO_UnitTestCase {
+
+	/** @var WPSEO_Admin_Editor_Specific_Replace_Vars_Test */
+	protected $class_instance;
+
+	/**
+	 * Set up the class which will be tested.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->class_instance = new WPSEO_Admin_Editor_Specific_Replace_Vars_Double();
+		// add_filter( 'wpseo_editor_specific_replace_vars', array( $this, 'filter_editor_specific_replacevars' ) );
```

Comment:
I think this line can be removed or maybe you can write a test for the filter. If you do that you should remove it after the assertion (`remove_filter( 'wpseo_editor_specific_replace_vars', array( $this, 'filter_editor_specific_replacevars' ) );`)